mixtera/utils/webdataset_utils.py

```diff
@@ -1,13 +1,12 @@
 import gzip
 import io
-from functools import partial
 from typing import Any, Iterator
 
 from wids.wids import group_by_key, splitname
 from wids.wids_mmtar import MMIndexedTar
 
 
-def decode(sample: dict[str, Any], decode_image: bool = True) -> dict[str, Any]:
+def decode_sample(sample: dict[str, Any]) -> dict[str, Any]:
```

Comment:
It's weird to have a funciton called `decode_sample` which you can optionally tell to not decode. Can you find a cleaner way to disable image decoding?